src/components/DialogContentText/DialogContentText.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { Meta } from '@storybook/react';
+import { withDesign } from 'storybook-addon-designs';
+
+import DialogContent, { DialogContentProps } from './DialogContentText';
+
+export const argTypes = {
+  children: {
```

Comment:
sometimes we use react node instead of text for this ? not sure if it's within scope of ticket to change it but it could be helpful!

Reply:
@eynglv kk, added it since I did use ReactNode in another Dialog story